core/src/main/java/org/bitcoinj/wallet/CoinSelection.java

```diff
@@ -53,4 +60,18 @@ private static Coin sumOutputValues(List<TransactionOutput> outputs) {
                 .map(TransactionOutput::getValue)
                 .reduce(Coin.ZERO, Coin::add);
     }
+
+    /**
+     * @return Total value of gathered outputs.
+     */
+    public Coin total() {
```

Comment:
`total` seems a bit vague to me. Maybe `totalValue` (or `totalAmount`)?

Reply:
`.totalValue()` sounds good. I'll change it.